numba/cuda/tests/doc_examples/ffi/functions.cu

```diff
@@ -0,0 +1,15 @@
+// Foreign function example: multiplication of a pair of floats
```

Comment:
I think this `.cu` file will need adding to `package_data` in `setup.py`.

Reply:
Added.